core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -59,12 +59,13 @@ def start_navigator(self) -> None:
         except Exception as error:
             raise RuntimeError(f"Failed to start navigator: {error}") from error
 
-        local_endpoint = "tcp:0.0.0.0:5760"
+        # ArduPilot process will connect as a client on the UDP server created by the mavlink router
+        master_endpoint = Endpoint("udpin", "127.0.0.1", 8852)
```

Comment:
Wait, FWIK ArduPilot only provides TCP endpoints, how are you using UDP here ?

Reply:
It's on the --help and it's working. Don't know if that was a recent change or not.

![image](https://user-images.githubusercontent.com/6551040/115718809-de2b7300-a351-11eb-9cbf-544672338eb3.png)

Reply:
Nice!